Workiva/over_react_codemod
# INTL-1608 : Activate _over_react_codemod executables separately

## Problem
  
AS per [Ticket](https://jira.atl.workiva.net/browse/INTL-1322) **INTL-1322** we need to run the sorting command with `ddev format `

## Solution
1. Create a sperate file in the Executable directory `lib/src/executables/sort_intl.dart`
2. Make **bin** file for the same    i.e. `bin/sort_intl.dart`

## How To QA
A. **Local system** : To check this in the Local system as debug mode do the following steps 

1. To activate `over_react_codemod` in local system as debug mode run the command `dart pub global activate --source path over_react_codemod`
2. To activate **executables** in the local system run command 
**dart pub global activate --source path . -x sort_intl**
this will show message on terminal 
```
Installed executable intl_sorting.
Activated over_react_codemod 2.26.0 at path
```
3. Also added `intl_sorting` as an executable in` pubspec.yaml`  [here](https://github.com/Workiva/over_react_codemod/blob/master/pubspec.yaml#L41)

4. Run `codemod` with executables only in any repo run this command 
`dart pub global run over_react_codemod: sort_intl `

B. As a global package  (This will work when this logic done as a Package in `pub.workiva`)
1. Activate `over_react_codemod`   run this command `dart pub global activate over_react_codemod`
2. Activate Executables `dart pub global activate over_react_codemod -x  sort_intl`
